extensions/azurecore/src/account-provider/azureAccountProviderService.ts

```diff
@@ -154,41 +154,25 @@ export class AzureAccountProviderService implements vscode.Disposable {
 
 			let simpleTokenCache = new SimpleTokenCache(tokenCacheKey, this._userStoragePath, noSystemKeychain, this._credentialProvider);
 			await simpleTokenCache.init();
-			const cachePath = path.join(this._userStoragePath, Constants.ConfigFilePath);
-
-			switch (platform) {
-				case Constants.Platform.Windows:
-					const dataProtectionScope = DataProtectionScope.CurrentUser;
-					const optionalEntropy = "";
-					this.persistence = await FilePersistenceWithDataProtection.create(cachePath, dataProtectionScope, optionalEntropy);
-					break;
-				case Constants.Platform.Mac:
-				case Constants.Platform.Linux:
-					this.persistence = await KeychainPersistence.create(cachePath, Constants.ServiceName, Constants.Account);
-					break;
-			}
-			if (!this.persistence) {
-				Logger.error('Unable to intialize persistence for access token cache. Tokens will not persist in system memory for future use.');
-				throw new Error('Unable to intialize persistence for access token cache. Tokens will not persist in system memory for future use.');
-			}
-
-			let persistenceCachePlugin: PersistenceCachePlugin = new PersistenceCachePlugin(
-				this.persistence, {
-				retryNumber: 500,
-				retryDelay: 150
-			});
 
-			const MSAL_CONFIG = {
+			const msalConfiguration: Configuration = {
 				auth: {
 					clientId: provider.metadata.settings.clientId,
-					redirect_uri: `${provider.metadata.settings.redirectUri}/redirect`
+					authority: 'https://login.windows.net/common'
+				},
+				system: {
+					loggerOptions: {
+						loggerCallback: this.getLoggerCallback(),
+						logLevel: MsalLogLevel.Verbose,
+						piiLoggingEnabled: false,
+					},
 				},
 				cache: {
-					cachePlugin: persistenceCachePlugin
+					cachePlugin: this._cachePluginProvider?.getCachePlugin()
```

Comment:
I would suggest adding a comment here explaining why you're using our own custom one (basically what you put in the description)

Reply:
Added here:
https://github.com/microsoft/azuredatastudio/pull/21335/files#diff-eb52caeb39b21d3d95a234f33f7f428ad4d42d60c6bdb6a5f839374811483d3cR90-R94